core/store/src/config.rs

```diff
@@ -174,6 +189,31 @@ impl StoreConfig {
             _ => bytesize::ByteSize::mib(32),
         }
     }
+
+    fn default_per_shard_max_bytes() -> HashMap<ShardUId, bytesize::ByteSize> {
+        let epoch_config_store = EpochConfigStore::for_chain_id(MAINNET, None).unwrap();
+        let mut shard_layouts: Vec<ShardLayout> = Vec::new();
+        // 72 is the latest protocol version of config store as of writing
+        for protocol_version in 72..=PROTOCOL_VERSION {
+            let epoch_config = epoch_config_store.get_config(protocol_version);
+            let shard_layout = epoch_config.shard_layout.clone();
+            // O(n) is fine as list is short
+            if !shard_layouts.contains(&shard_layout) {
+                shard_layouts.push(shard_layout);
+            }
+        }
+
+        let mut per_shard_max_bytes: HashMap<ShardUId, bytesize::ByteSize> = HashMap::new();
+        for (shard_uid, bytes) in PER_ACCOUNT_CACHE_SIZE.iter() {
+            let account_id = AccountId::from_str(shard_uid)
+                .expect("the hardcoded account id should guarantee to be valid");
+            for shard_layout in shard_layouts.iter() {
+                let shard_uid = account_id_to_shard_uid(&account_id, &shard_layout);
```

Comment:
nit: please prefer the ShardLayout::account_id_to_shard_uid

Reply:
I think Wac is referring to the current function you're using. Feel free to ignore this comment.